test/test-lint.cpp

```diff
@@ -1247,9 +1261,9 @@ TEST(test_lint, use_variable_declared_in_grandparent_function) {
 }
 
 TEST(test_lint, use_for_loop_let_variable_before_or_after_loop) {
+  const char8 use_after[] = u8"element";
```

Comment:
I made these ordering changes because the address of these arrays is decreasing from top to bottom.